spec/spec_helper.rb

```diff
@@ -84,6 +84,7 @@ def photo_fixture_name
     $process_queue = false
     allow_any_instance_of(Postzord::Dispatcher::Public).to receive(:deliver_to_remote)
     allow_any_instance_of(Postzord::Dispatcher::Private).to receive(:deliver_to_remote)
+    load "#{Rails.root}/db/seeds.rb"
```

Comment:
See above for discussion about  the Scope model. If we decide to keep it the proper solution is to change the Travis script to call the seeds file.